pkg/lib/editor/js/core-config-setup/core-config-setup.js

```diff
@@ -287,6 +287,15 @@ angular.module("quay-config")
           if (!("LOG_ARCHIVE_LOCATION" in config)) {
             config["LOG_ARCHIVE_LOCATION"] = "default";
           }
+          if (!("ALLOWED_OCI_ARTIFACT_TYPES" in config)){
+            config["ALLOWED_OCI_ARTIFACT_TYPES"] = {}
+            config["ALLOWED_OCI_ARTIFACT_TYPES"][
+              "application/vnd.oci.image.config.v1+json"
+            ] = ["application/vnd.oci.image.layer.v1.tar+zstd"];
+            config["ALLOWED_OCI_ARTIFACT_TYPES"][
+              "application/vnd.sylabs.sif.config.v1+json"
+            ] = ["application/vnd.sylabs.sif.layer.v1+tar"];
```

Comment:
Idk that we want to make that default.
We probably want to make helm a default one though, and slowly deprecate the `HELM_CHART_CONFIG_TYPE` and `HELM_CHART_LAYER_TYPES` flags that we have.
`"application/vnd.cncf.helm.config.v1+json": ["application/tar+gzip"]`

Reply:
Otherwise LGTM